emukit/core/parameter.py

```diff
@@ -21,6 +21,13 @@ def model_parameters(self) -> List:
         """
         return [self]
 
+    @property
+    def bounds(self) -> List[Tuple]:
+        """
+        Returns the minimum and maximum values for the encoded version of the parameter
```

Comment:
I'd suggest just "Returns bounds of the parameter in a form of list of tuples" or something like that. Mentioning encodings here might confuse people, as there is no encoding taking place for most parameters

Reply:
Changed